gradle-jdks/src/test/groovy/com/palantir/gradle/jdks/JdksPluginIntegrationSpec.groovy

```diff
@@ -53,21 +58,16 @@ class JdksPluginIntegrationSpec extends IntegrationSpec {
 
             public final class PrintJavaVersion {
                 public static void main(String... args) {
-                    System.out.println(String.format(
-                            "version: %s, vendor: %s",
+                    System.out.printf(
+                            "version: %s, vendor: %s%n",
                             System.getProperty("java.version"),
-                            System.getProperty("java.vendor")));
+                            System.getProperty("java.vendor"));
                 }
             }
-        '''.stripIndent(true)
+        '''.stripIndent(true), subprojectDir
 
         when:
-
-        def stdout = runTasksSuccessfully(
-                'printJavaVersion',
-                // TODO: avoid resolving from root configuration somehow, removed in Gradle 8
-                '--warning-mode=none')
```

Comment:
I really thought I encountered this deprecation warning - but seemingly I can't get it to reappear, even using a subproject 🤷🏻